### PR TITLE
add mutect 1.1.5 module

### PR DESCRIPTION
**Pull request by huyu335**
Tested on SGE and LSF

### CODE REVIEW

**Comment by rhshah** [comment]
Thank you @huyu335, can we please add `README.md` for the tool

**Comment by huyu335** [comment]
Hi @rhshah, I have added the README file in. Thank you!

**Comment by rhshah** [comment]
Thank you @huyu335, I see that we have not uploaded the docker image to docker hub, nor we have referenced it in the CWL, here is what I did:
```
$ cd mutect_1.1.5/container
$ docker build --no-cache=true --build-arg BUILD_DATE=$(date -u +'%Y-%m-%dT%H:%M:%SZ') --build-arg BUILD_VERSION="0.1.0" -t mskcc/mutect:0.1.0 .
$ docker push mskcc/mutect:0.1.0
```
I have made the cwl modification using Rabix thus you might see more changes but, can you please test this version:
- [] cwltool
- [] toil-cwl-runner (LSF)

**Comment by huyu335** [comment]
Hi @rhshah, I just tested it with both cwltool and toil-cwl-runner. All succeeded. Can you please check again? Thank you!

File: mutect_1.1.5/mutect_1.1.5.cwl [hunk 1]
```diff
@@ -656,7 +662,7 @@ outputs:
         ${
           if (inputs.out)
             return inputs.out;
-          return null;
+          return 'null';
         }
```
Comment (rhshah) [inline mutect_1.1.5/mutect_1.1.5.cwl]: Do we need to specify return null?

Comment (huyu335) [reply mutect_1.1.5/mutect_1.1.5.cwl]: Hi @rhshah I just modified the output file names in my latest commit. Can you please try again? Thank you!

**Pull request merged**